packages/tracking/src/trackEvent.ts

```diff
@@ -21,6 +21,18 @@ import { config } from './shared';
 import isTrackingEnabled from './isTrackingEnabled';
 import track from './track';
 
+interface EventParameters {
+  name?: string;
```

Comment:
Should any of these be required?

Reply:
Perhaps in the future it might make sense to divide this into multiple interfaces, which on their own could have some required params. For now I think this suffices.